packages/venia-concept/src/components/MyAccountMenuPage/MenuItem/menuItem.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classify from 'src/classify';
+import defaultClasses from './menuItem.css';
+
+class MenuItem extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            item: PropTypes.string
+        }),
+        component: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+        title: PropTypes.node,
+        rightNode: PropTypes.node
```

Comment:
This is a bad prop name. What is a `rightNode`?

Reply:
@jimbo We need this to render badges and some other additional information near the title of some items https://user-images.githubusercontent.com/2686596/46780264-fef3c600-cd1b-11e8-9098-d53b11ce8ddc.png. `rightNode` can be any element displayed to the right of the title.

Reply:
Can we call this a "badge"?

Reply:
Sure. Something semantic that's not tied to an arbitrary aspect of its presentation is what we should aim for. Could be `badge`, `subtext`, etc.